<launch>

    <arg name="enable" default="false" />

    <node ns="led" pkg="ros_ws281x" name="ros_ws281x_node" type="ros_ws281x_node" clear_params="true" output="screen" if="$(arg enable)" required="false">
        <param name="target_frequency" value="800000"/>
        <param name="gpio_pin" value="18"/>
        <param name="dma" value="10"/>
        <param name="strip_type" value="WS2811_STRIP_GRB"/>
        <param name="led_count" value="60"/>
        <param name="invert" value="false"/>
        <param name="brightness" value="64"/>
    </node>
</launch>
